sumpy/e2e.py

```diff
@@ -791,8 +791,8 @@ def get_kernel(self, result_dtype):
                 for itgt_box
                     [itgt_coeff]: tgt_expansions[itgt_box, itgt_coeff] = \
                         m2l_postprocess_inner(
-                            tgt_rscale,
                             src_rscale,
+                            tgt_rscale,
```

Comment:
loopy doesn't have keyword arguments right?

Reply:
:sob: No. There are some rudiments, but it didn't ultimately get done.